projects/ifa-dissertation/js/index.js

```diff
@@ -182,48 +178,48 @@ d3.csv("./data/ifa-dissertations.csv", function(error, data) {
       "text-anchor": "middle",
       fill: "white"
     });
+
   function tick(e) {
-    node
-      .each(gravity(.2 * e.alpha))
-      .each(collide(.5))
-      .attr("cx", function (d) {return d.x;})
+    nodes
+      .each(gravity(.1 * e.alpha))
+      .each(collide(0.5))
+      .attr("cx", function (d) {
+        return d.x;
+      })
       .attr("cy", function (d) {return d.y;});
   }
 
   function gravity(alpha) {
     return function (d) {
-      d.y += (d.cy - d.y) * alpha;
-      d.x += (d.cx - d.x) * alpha;
+      d.y += (d.centerY - d.y) * alpha;
+      d.x += (d.centerX - d.x) * alpha;
```

Comment:
Lerp `d.x, d.y` to `d.centerX, d.centerY` so all nodes are attracted towards the center